layers/+completion/helm/packages.el

```diff
@@ -216,57 +216,37 @@
         ;; helm-ag marks
         "s`"  'helm-ag-pop-stack
-        ;; opened buffers scope
-        "sb"  'spacemacs/helm-buffers-smart-do-search
-        "sB"  'spacemacs/helm-buffers-smart-do-search-region-or-symbol
-        "sab" 'helm-do-ag-buffers
-        "saB" 'spacemacs/helm-buffers-do-ag-region-or-symbol
-        "skb" 'spacemacs/helm-buffers-do-ack
-        "skB" 'spacemacs/helm-buffers-do-ack-region-or-symbol
-        "srb" 'spacemacs/helm-buffers-do-rg
-        "srB" 'spacemacs/helm-buffers-do-rg-region-or-symbol
-        "stb" 'spacemacs/helm-buffers-do-pt
-        "stB" 'spacemacs/helm-buffers-do-pt-region-or-symbol
-        ;; current file scope
-        "ss"  'spacemacs/helm-file-smart-do-search
-        "sS"  'spacemacs/helm-file-smart-do-search-region-or-symbol
-        "saa" 'helm-ag-this-file
-        "saA" 'spacemacs/helm-file-do-ag-region-or-symbol
-        ;; files scope
-        "sf"  'spacemacs/helm-files-smart-do-search
-        "sF"  'spacemacs/helm-files-smart-do-search-region-or-symbol
-        "saf" 'helm-do-ag
-        "saF" 'spacemacs/helm-files-do-ag-region-or-symbol
-        "skf" 'spacemacs/helm-files-do-ack
-        "skF" 'spacemacs/helm-files-do-ack-region-or-symbol
-        "srf" 'spacemacs/helm-files-do-rg
-        "srF" 'spacemacs/helm-files-do-rg-region-or-symbol
-        "stf" 'spacemacs/helm-files-do-pt
-        "stF" 'spacemacs/helm-files-do-pt-region-or-symbol
-        ;; current dir scope
-        "sd"  'spacemacs/helm-dir-smart-do-search
-        "sD"  'spacemacs/helm-dir-smart-do-search-region-or-symbol
-        "sad" 'spacemacs/helm-dir-do-ag
-        "saD" 'spacemacs/helm-dir-do-ag-region-or-symbol
-        "skd" 'spacemacs/helm-dir-do-ack
-        "skD" 'spacemacs/helm-dir-do-ack-region-or-symbol
-        "srd" 'spacemacs/helm-dir-do-rg
-        "srD" 'spacemacs/helm-dir-do-rg-region-or-symbol
-        "std" 'spacemacs/helm-dir-do-pt
-        "stD" 'spacemacs/helm-dir-do-pt-region-or-symbol
-        ;; current project scope
-        "/"   'spacemacs/helm-project-smart-do-search
-        "*"   'spacemacs/helm-project-smart-do-search-region-or-symbol
-        "sp"  'spacemacs/helm-project-smart-do-search
-        "sP"  'spacemacs/helm-project-smart-do-search-region-or-symbol
-        "sap" 'spacemacs/helm-project-do-ag
-        "saP" 'spacemacs/helm-project-do-ag-region-or-symbol
-        "skp" 'spacemacs/helm-project-do-ack
-        "skP" 'spacemacs/helm-project-do-ack-region-or-symbol
-        "srp" 'spacemacs/helm-project-do-rg
-        "srP" 'spacemacs/helm-project-do-rg-region-or-symbol
-        "stp" 'spacemacs/helm-project-do-pt
-        "stP" 'spacemacs/helm-project-do-pt-region-or-symbol))
+        ;; buffer(s) scope
+        "sb"  'spacemacs/helm-smart-search-buffer
+        "sB"  'spacemacs/helm-smart-search-buffer-with-input
+        ;; directory scope
+        "sd"  'spacemacs/helm-smart-search-dir
+        "sD"  'spacemacs/helm-smart-search-dir-with-input
```

Comment:
These functions cannot be found, I think this is a direct side effect of the not executed "--each" block but nevertheless please double check.